packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+
+interface ContentProps {
+	title: string;
+	featuredImage: string;
+	content: string;
+	imgUrl: string;
+	date?: string;
+}
+
+/**
+ *
+ * @param props - The props needed for the ContentWithImage component
+ * @param props.title - The title of your content
+ * @param props.featuredImage - A URL for the image source of your content
+ * @param props.content - A string of elements to make up the content of your post
+ * @param props.imgUrl - The image URL coming from your enviornment config
```

Comment:
This description can be a bit more specific and maybe could use a link to the docs on `next/image`